packages/pulpcore-obsolete-packages/pulpcore-obsolete-packages.spec

```diff
@@ -0,0 +1,32 @@
+Name: foreman-obsolete-packages
+Version: 1.0
+Release: 1%{?dist}
+License: MIT
+Summary: A package to obsolete retired packages
+URL: https://github.com/theforeman/pulpcore-packaging
+BuildArch: noarch
+
+%if 0%{?!scl:1}
```

Comment:
let's remove the scl content, we don't need that anymore, the only if necessary is the one for `rhel8`